apps/rush-lib/src/start.ts

```diff
@@ -1,30 +1,4 @@
-// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
-// See LICENSE in the project root for license information.
-
-import { EOL } from 'os';
-import * as colors from 'colors';
-
+import { start } from './cli/start';
 import rushVersion from './rushVersion';
-import RushCommandLineParser from './cli/actions/RushCommandLineParser';
-
-/**
- * Executes the Rush CLI. This is expected to be called by the @microsoft/rush package, which acts as a version manager
- *  for the Rush tool. The rush-lib API is exposed through the index.ts/js file.
- *
- * @param launcherVersion The version of the @microsoft/rush wrapper used to call invoke the CLI.
- * @param isManaged True if the tool was invoked from within a project with a rush.json file, otherwise false. We
- *  consider a project without a rush.json to be "unmanaged" and we'll print that to the command line when
- *  the tool is executed. This is mainly used for debugging purposes.
- */
-export function start(launcherVersion: string, isManaged: boolean): void {
-  console.log(
-    EOL +
-    colors.bold(`Rush Multi-Package Build Tool ${rushVersion}` + colors.yellow(isManaged ? '' : ' (unmanaged)')) +
-    colors.cyan(' - http://aka.ms/rush') +
-    EOL
-  );
-
-  const parser: RushCommandLineParser = new RushCommandLineParser();
 
-  parser.execute();
-}
+start(rushVersion, false);
```

Comment:
>start [](start = 0, length = 5)

"start.js" is our standard entry point for executable binaries.

Maybe rename this file to debug.js and add a comment like this:

```
// To facilitate debugging, this entry point is useful
// for invoking the rush-lib backend directly from a debugger.
```

Reply:
This is a standard entrypoint, though. You can execute it if you want and it'll run rush

---
In reply to: [147528906](https://github.com/Microsoft/web-build-tools/pull/417#discussion_r147528906) [](ancestors = 147528906)